cms/models.py

```diff
@@ -186,3 +188,38 @@ class CoursePage(ProductPage):
     )
 
     content_panels = [FieldPanel("course")] + ProductPage.content_panels
+
+
+class FrequentlyAskedQuestionPage(CourseProgramChildPage):
+    """
+    FAQs page for program/course
+    """
+    content_panels = [
+        InlinePanel('faqs', label='Frequently Asked Questions'),
+    ]
+
+    def save(self, *args, **kwargs):
+        # autogenerate a unique slug so we don't hit a ValidationError
+        self.title = "Frequently Asked Questions"
+        self.slug = slugify("{}-{}".format(self.get_parent().id, self.title))
+        super(FrequentlyAskedQuestionPage, self).save(*args, **kwargs)
```

Comment:
We are following now. 
`super().save(*args, **kwargs)`

Reply:
done